Packs/Code42/Integrations/Code42/Code42_test.py

```diff
@@ -1380,14 +1410,34 @@ def test_departingemployee_get_all_command_when_no_employees(
         no_employees_response
     )
     client = create_client(code42_departing_employee_mock)
-    cmd_res = departingemployee_get_all_command(client,{})
+    cmd_res = departingemployee_get_all_command(client, {})
     assert cmd_res.outputs_prefix == "Code42.DepartingEmployee"
     assert cmd_res.outputs_key_field == "UserID"
     assert cmd_res.raw_response == {}
     assert cmd_res.outputs == {"Results": []}
     assert code42_departing_employee_mock.detectionlists.departing_employee.get_all.call_count == 1
 
 
+def test_departingemployee_get_command(code42_departing_employee_get_mock):
+    client = create_client(code42_departing_employee_get_mock)
+    cmd_res = departingemployee_get_command(
+        client, {"username": _TEST_USERNAME}
+    )
+    get_func = code42_departing_employee_get_mock.detectionlists.departing_employee.get
+    get_func.assert_called_once_with(_TEST_USER_ID)
+    assert cmd_res.raw_response == _TEST_USERNAME
```

Comment:
Make sure to test the outputs as well as the raw response. You don't have to test human readable outputs though.

Reply:
Done.